### ISSUE TITLE
DSBugBash: "Sort Imports" shouldn't move the #%% line with the import statement

### ISSUE THREAD

**Issue by ronglums** [microsoft/vscode-jupyter#3840]
1. Put the following content into a py file.
import pandas as pd # pandas is a dataframe library
import matplotlib.pyplot as plt # matplotlib.pyplot plots data
import numpy as np # numpy provides N-dim object support

#%% [markdown]
# ## Load and review data

#%%
from sklearn.cross_validation import train_test_split

2. Right click in the editor and select "Sort Imports". 

Expected:
only this line "from sklearn.cross_validation import train_test_split" gets moved to the top

Actual:
The "#%%" line above "from sklearn.cross_validation import train_test_split" was also moved.
